sync/class.jetpack-sync-defaults.php

```diff
@@ -129,6 +129,8 @@ class Jetpack_Sync_Defaults {
 		'mailserver_login', // Not syncing contents, only the option name
 		'mailserver_pass', // Not syncing contents, only the option name
 		'mailserver_port',
+		'google_my_business_keyring_id',
```

Comment:
We should be prefixing these options with jetpack_

Reply:
Do we need to actually name those `jetpack_google_my_business_keyring_id` and `jetpack_google_my_business_location_id` and update all usage of those props in wpcom and calypso as well? Or should we use `Jetpack_Options::update_option` and `Jetpack_Options::get_option` so they get the prefix automatically when inserting/retrieving from the database?

@enejb I'm not sure I understand, could you explain a bit how those two play together?